libraries/AP_WindVane/AP_WindVane.h

```diff
@@ -47,10 +47,10 @@ class AP_WindVane
     void update();
 
     // get the apparent wind direction in radians
-    float get_apparent_wind_direction_rad();
+    float get_apparent_wind_direction_rad(void);
```

Comment:
I'm not a fan of adding the "void" actually.. there are various opinions in the group but..

Reply:
@rmackay9 Hey, I was just curious whist reading about this PR. What is an advantage of not adding the 'void' explicitly? Is it more readable- or such?